a10_neutron_lbaas/db/models.py

```diff
@@ -53,24 +59,47 @@ def find_all(cls, db_session=None):
         return cls.query(db_session).all()
 
     @classmethod
+    def create(cls, **kwargs):
```

Comment:
No, because I need to make models that don't have all those columns in a10-openstack.